OWNERS_ALIASES

```diff
@@ -2,19 +2,20 @@ aliases:
   core-approvers:
   - abhat
   - danwinship
-  - dougbtv
   - fepan
   - JacobTanenbaum
   - jcaamano
   - knobunc
   - kyrtapz
-  - trozet
+  - pliurh
+  - tssurya
   core-reviewers:
   - abhat
+  - bpickard22
   - danwinship
   - dougbtv
```

Comment:
@jcaamano in reviewers list let's add the full team

also @JacobTanenbaum do you get cycles to do CNO reviews - would you rather be in approvers list than in reviewers? cause then you'd be assigned to review PRs for a +1..

let's add all our team Jaime. @arkadeepsen @arghosh93 @martinkennelly @pperiyasamy @ricky-rav as +1 reviewers @miheer as well

Reply:
Right, had actually forgotten what we decided. 

The bot probably picks on of each list right? It does add two reviewers.